### PR TITLE
Enable metrics logs

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

Ref: https://issues.redhat.com/browse/ACM-9125

### CODE REVIEW

File: manager/pkg/cronjob/task/local_compliance_history.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ func SyncLocalCompliance(ctx context.Context, enableSimulation bool, job gocron.
 		log.Error(err, "sync from local_status.compliance to history.local_compliance failed")
 		return
 	}
-	log.V(2).Info("with local_status.compliance", "totalCount", statusTotal, "insertedCount", statusInsert)
+	log.Info("with local_status.compliance", "totalCount", statusTotal, "insertedCount", statusInsert)
```
Comment (clyang82) [inline manager/pkg/cronjob/task/local_compliance_history.go:78]: Can we pass `-v=2` to enable it?

Comment (yanmxa) [reply manager/pkg/cronjob/task/local_compliance_history.go:78]: But I mean to enable it at the information level, cause the cronjob only executes once a day, It won't produce much log but it's helpful for troubleshooting!

Comment (clyang82) [reply manager/pkg/cronjob/task/local_compliance_history.go:78]: but you can get such information in database, correct?

Comment (yanmxa) [reply manager/pkg/cronjob/task/local_compliance_history.go]: modify the log level to V(2)

**Pull request merged**